packager/media/formats/mp4/fragmenter.cc

```diff
@@ -196,26 +196,38 @@ Status Fragmenter::FinalizeFragment() {
     traf_->sample_to_groups.resize(traf_->sample_to_groups.size() + 1);
     SampleToGroup& sample_to_group = traf_->sample_to_groups.back();
     sample_to_group.grouping_type = FOURCC_roll;
-
+    
+    
     sample_to_group.entries.resize(1);
     SampleToGroupEntry& sample_to_group_entry = sample_to_group.entries.back();
     sample_to_group_entry.sample_count = traf_->runs[0].sample_count;
     sample_to_group_entry.group_description_index =
         SampleToGroupEntry::kTrackGroupDescriptionIndexBase + 1;
+        
+
   }
   for (const auto& sample_group_description :
        traf_->sample_group_descriptions) {
     traf_->sample_to_groups.resize(traf_->sample_to_groups.size() + 1);
     SampleToGroup& sample_to_group = traf_->sample_to_groups.back();
     sample_to_group.grouping_type = sample_group_description.grouping_type;
-
+    
     sample_to_group.entries.resize(1);
     SampleToGroupEntry& sample_to_group_entry = sample_to_group.entries.back();
     sample_to_group_entry.sample_count = traf_->runs[0].sample_count;
     sample_to_group_entry.group_description_index =
         SampleToGroupEntry::kTrackFragmentGroupDescriptionIndexBase + 1;
   }
 
```

Comment:
I assume that this file is added by mistake. Can you remove it from this change? There may be other files added by mistake too.